schema/src/main/scala/io/shiftleft/codepropertygraph/schema/CallGraph.scala

```diff
@@ -115,15 +117,15 @@ object CallGraph extends SchemaBase {
     val dispatchType = builder
       .addProperty(
         name = "DISPATCH_TYPE",
-        valueType = ValueTypes.STRING,
-        cardinality = Cardinality.One,
+        valueType = ValueType.String,
         comment = """This field holds the dispatch type of a call, which is either `STATIC_DISPATCH` or
             |`DYNAMIC_DISPATCH`. For statically dispatched method calls, the call target is known
             |at compile time while for dynamically dispatched calls, it can only be determined at
             |runtime as it may depend on the type of an object (as is the case for virtual method
             |calls) or calculation of an offset.
             |""".stripMargin
       )
+      .mandatory(PropertyDefaults.String)
```

Comment:
We could consider either STATIC_DISPATCH or DYNAMIC_DISPATCH as default

Reply:
@fabsx00 @ml86 wdyt?

Reply:
Hard to say. There are languages such as C, where STATIC_DISPATCH would be a good default, and scripting languages where DYNAMIC_DISPATCH is. So, I think I'd go with not giving this a default.

Reply:
I think we are fine with the default default.

Reply:
:+1: